src/session/get-access-token.ts

```diff
@@ -120,6 +150,12 @@ export default function accessTokenFactory(
         user: { ...session.user, ...newSession.user }
       });
 
+      if (accessTokenRequest?.afterRefresh) {
+        session = await accessTokenRequest.afterRefresh(req as NextApiRequest, res as NextApiResponse, session);
+      }
+
+      sessionCache.set(req, res, session as Session);
```

Comment:
Why does this require a cast, given that successfully awaiting `afterRefresh` produces a Session?

Reply:
`sessionCache.get` can return `undefined` (if a session hasn't been estabilished) and TS isn't smart enough to know that there is a session if `client.refresh` is successful

Reply:
Actually, my bad, I don;t need to cast it - will fix, thanks

Reply:
fixed c80ea310f0f44df75cc85e8272debbd0661a78b7